portal-api/src/v1/controllers/eligibilityCriteria.controller.js

```diff
@@ -21,7 +21,7 @@ exports.findEligibilityCriteria = findEligibilityCriteria;
 
 const findOneEligibilityCriteria = async (version, callback) => {
   const collection = await db.getCollection('eligibilityCriteria');
-  collection.findOne({ version }, (error, result) => {
+  collection.findOne({ version: { $eq: version } }, (error, result) => {
```

Comment:
I'm pretty sure this should be `typeof version === 'number` validation (which I've added) because it gets called here:
`exports.findOne = (req, res) => (
  findOneEligibilityCriteria(
    Number(req.params.version),
    (eligibilityCriteria) => res.status(200).send(eligibilityCriteria),
  )
);`